iothub/service/src/AmqpServiceClient.cs

```diff
@@ -115,16 +119,15 @@ public async override Task SendAsync(string deviceId, Message message, TimeSpan?
             try
             {
                 SendingAmqpLink sendingLink = await GetSendingLinkAsync().ConfigureAwait(false);
+                Outcome outcome = await sendingLink
+                    .SendMessageAsync(amqpMessage, IotHubConnection.GetNextDeliveryTag(ref _sendingDeliveryTag), AmqpConstants.NullBinary, timeout.Value)
+                    .ConfigureAwait(false);
 
                 if (Logging.IsEnabled)
                 {
-                    Logging.Info(this, $"Retrieved SendingAmqpLink [{sendingLink.Name}]", nameof(SendAsync));
+                    Logging.Info(this, $"Outcome was: {outcome?.DescriptorName}", nameof(SendAsync));
```

Comment:
FWIW, I find when a field can be empty or null, it can be hard to notice that when the statement is logged. That is why I like to surround my values with square brackets, although some people prefer single quote marks. You might consider that here.

Reply:
Surprisingly our logger takes care of that: https://github.com/Azure/azure-iot-sdk-csharp/blob/master/common/src/Logging.Common.cs#L44